src/webgl-preview.ts

```diff
@@ -403,13 +304,20 @@ export class WebGLPreview {
     this.initScene();
 
     this.renderLayerIndex = 0;
+
+    if (this.job.layers() === null) {
```

Comment:
I believe you're executing `job.layers()` multiple times within a single render pass. You prob want to avoid that by just keep a local variable or instance prop _layers or something.

Reply:
This is fixed in https://github.com/remcoder/gcode-preview/pull/220 🎉

Reply:
ok nvrnd, I've seen the indexers PR now :-)